tests/model/PaginatedListTest.php

```diff
@@ -325,7 +325,8 @@ public function testPrevLink() {
 
 		$this->assertNull($list->PrevLink());
 		$list->setCurrentPage(2);
-		$this->assertContains('start=0', $list->PrevLink());
+		$this->assertNotNull($list->PrevLink());
+		$this->assertNotContains('start=0', $list->PrevLink());
```

Comment:
Please add a test where other query params are present so we can ensure there are no regressions